desc/profiles.py

```diff
@@ -75,7 +75,7 @@ def __repr__(self):
             type(self).__name__
             + " at "
             + str(hex(id(self)))
-            + " (name={}, grid={})".format(self.name, self.grid)
+            + " (name={}, grid={})".format(self._name, self._grid)
```

Comment:
@ddudt Shouldn't these be referencing the property, not the underlying attribute? Same in the rest of this file